### ISSUE TITLE
URL

### ISSUE THREAD

**Issue by keep-learning-go** [kgretzky/evilginx2#22]
Hey @kgretzky , thank you man, you've done really great job!
It is not an actually "issue" , I just have one question.
Is there any way to generate "clean" links, without base64 extension?
If there is no short answer, please just point me to the related files, so I'll inspect and modify them. Thanks again!

**Comment by kgretzky** [comment]
Hey!

You mean to generate links without the parameter that holds the redirection URL or you want the value of the redirection parameter to be in clear text without base64 encoding?

**Comment by keep-learning-go** [comment]
Yes, without the parameter that holds the redirection URL at all.
Is there any way to make it "optional" ?
Redirection is not needed in my case.
Thank you.

**Comment by kgretzky** [comment]
Yes, you can pass an empty redirection URL using empty quotes `""`:
```
phishlets get-url <phishlet> ""
```

**Comment by keep-learning-go** [comment]
Alright, close but, still not exactly what I'm trying to achieve.
Extension now got shorter ( "" = &xt=fe98 )
What I'm trying achieve is to disable "unauthorized request", so that my link can be:
www.my-phishlet-link.com/login (without any kind of b64 extension or whatsover).

**Comment by keep-learning-go** [comment]
I found a "workaround" for this. I replaced hardcoded Rick YouTube url to my "gateway" url. So anytime there is unauthorized visit to www.my-phishlet-link.com/login it will redirected to www.my-phishlet-link.com/login&xt=fe98. 
Thank you Kuba for such amazing tool once again!
Also, I wanted you to have a look at this nice tool:
https://github.com/boxug/trape
maybe in the future you or some Gitters here, will merge these 2 projects. It has nice control panel elements. Good luck.
